src/domain/opendatafabric/src/identity/dataset_refs.rs

```diff
@@ -739,79 +739,117 @@ impl std::cmp::PartialOrd for DatasetRefAny {
     }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// DatasetRefPattern
+////////////////////////////////////////////////////////////////////////////////
+
 #[derive(Debug, Clone, Eq, PartialEq)]
-pub enum DatasetRefPattern {
+pub enum DatasetRefPatternLocal {
```

Comment:
The old name seemed more consistent with the rest of code:
- `DatasetRef` - local reference
- `DatasetRefRemote` - remote reference